extensions/amp-youtube/validator-amp-youtube.protoascii

```diff
@@ -29,6 +29,7 @@ tags: {  # <amp-youtube>
   tag_name: "AMP-YOUTUBE"
   requires_extension: "amp-youtube"
   attrs: { name: "autoplay" }
+  attrs: { name: "credentials" }
```

Comment:
Please restrict the potential values to only those used by the extension.

```
attrs: {
  name: "credentials"
  value_regex_casei: "(include|omit)"
}
```

Reply:
I wanted to add that but I didn't know how since the "credentials" in <amp-list> also do not define that. :)